src/fsutil.rs

```diff
@@ -0,0 +1,15 @@
+use std::{fs, io, path::Path};
+
+pub(crate) fn copy_dir_all(src: impl AsRef<Path>, dst: impl AsRef<Path>) -> io::Result<()> {
+    fs::create_dir_all(&dst)?;
+    for entry in fs::read_dir(src)? {
+        let entry = entry?;
+        let ty = entry.file_type()?;
+        if ty.is_dir() {
```

Comment:
have you tried what happens with symlinks that are pointing to a parent folder?

Reply:
It'll fail during copy - `is_dir()` is mutually exclusive with `is_file` and `is_symbolic`, so it'll attempt to copy it as a file.

`fs::copy` fails when the source is not a file...

Reply:
So symbolic links inside the template is not currently supported